packages/core/src/Stores/Helpers/chart-marker-helpers.js

```diff
@@ -127,7 +129,7 @@ export const createMarkerSpotMiddle = (contract_info, tick, idx) => {
         spot_value: `${spot}`,
         spot_epoch,
         align_label: tick.align_label,
-        is_value_hidden: is_accumulator,
+        is_value_hidden: is_accumulator || (isOnlyUpsDownsContract(contract_info.contract_type) && idx !== 1),
```

Comment:
According to new requirements we should hide information about tick (value, time) for all ticks, except the first and the last